server/controllers/routes/login.js

```diff
@@ -0,0 +1,33 @@
+const bcrypt = require('bcrypt');
+const jwt = require('jsonwebtoken');
+require('env2')('./secret.env');
+const { loginData } = require('../../database/quieres/login');
+
+const secret = process.env.SECRET_KEY;
+
+const login = (req, res, next) => {
+  const { email, password } = req.body;
+  loginData(email)
+    .then((result) => {
+      if (result.rows.length === 0) res.send('not signed up');
```

Comment:
server talks to web app by **JSON**
```suggestion
      if (result.rows.length === 0) res.status(401).send({
    error: "unauthorized "
    "Status-Code": 401
    });
```